styles/widgets/material/button.material.less

```diff
@@ -199,7 +195,6 @@
 }
 
 .dx-button { // stylelint-disable-line no-duplicate-selectors
-    height: @MATERIAL_BUTTON_HEIGHT;
```

Comment:
Default button height must be 36px by design

Reply:
fixed